README.md

```diff
@@ -36,8 +36,12 @@ export default defineConfig({
 ```json
 // tsconfig.json
 "compilerOptions": {
-  // ...
-  "types": ["vite/client", "vite-plugin-solid-svg/types"],
+  "types": [
+    "vite/client",
+    "vite-plugin-solid-svg/types",
+    // Use vite-plugin-solid-svg/types-default together with `defaultAsComponent`
```

Comment:
Maybe `vite-plugin-solid-svg/as-component`?

Reply:
Sounds nice, but for consistency will use `component-solid`.